cpp/src/arrow/flight/flight_test.cc

```diff
@@ -1105,6 +1110,15 @@ TEST_F(TestFlightClient, DoGetDicts) {
   CheckDoGet(descr, expected_batches, check_endpoints);
 }
 
+// Ensure the gRPC client is configured to allow large messages
+// Tests a 32 MiB batch
+TEST_F(TestFlightClient, DoGetLargeBatch) {
+  BatchVector expected_batches;
+  ASSERT_OK(ExampleLargeBatches(&expected_batches));
+  Ticket ticket{"ticket-large-batch-1"};
+  CheckTicket(ticket, expected_batches);
```

Comment:
I was a bit confused here because I didn't understand that `CheckTicket` actually issued a GET request. Can you call it `CheckDoGet` instead?